config.yaml.bak

```diff
@@ -1,3 +1,6 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
 options:
```

Comment:
`s/config.yaml.bak/config.yaml/` ?

Reply:
I hear you, this is just a temporary store for config before I add a PR that actually uses it, likely the next!